addons/mail/static/src/components/discuss/tests/discuss_tests.js

```diff
@@ -1284,6 +1284,34 @@ QUnit.test('basic rendering of message', async function (assert) {
     );
 });
 
+QUnit.test('should not be able to reply to temporary/transient messages', async function (assert) {
+    assert.expect(1);
+
+    this.data['mail.channel'].records.push({ id: 20 });
+    await this.start({
+        discuss: {
+            params: {
+                default_active_id: 'mail.channel_20',
+            },
+        },
+    });
+    //These user interactions is to forge a transient message response from channel command "/who"
+    await afterNextRender(() => {
+        document.querySelector(`.o_ComposerTextInput_textarea`).focus();
+        document.execCommand('insertText', false, "/who");
+    });
+    //Click on message to show actions on the transient message resulting from the "/who" command
+    await afterNextRender(() =>
+        document.querySelector('.o_Composer_buttonSend').click()
+    );
+    await afterNextRender(() => document.querySelector('.o_Message').click());
```

Comment:
The comment should be above the click on `.o_Message`, not `.o_Composer_buttonSend` (which is a user interaction related to forging the transient message response).